src/main/java/com/owczarczak/footballers/footballer/FootballerRepository.java

```diff
@@ -6,21 +6,13 @@
 
 import java.util.List;
 
-public interface FootballerRepository extends JpaRepository<Footballer, Integer> {
-    @Query(value = """
-            select *
-            from footballers
-            order by height desc
-            limit 3
-            """, nativeQuery = true)
-    List<Footballer> get3HighestFootballers();
+public interface FootballerRepository extends JpaRepository<FootballerDto, Integer> {
 
-//    @Query(value = """
-//            select *
-//            from footballers
-//            where 'name' like '%search%'
-//            """, nativeQuery = true)
-//    List<Footballer> getFootballersByName(@Param("search") String name);
+    @Query("select * from footballers order by height desc limit 3")
+    List<FootballerDto> get3HighestFootballers();
```

Comment:
Repozytorium działa na encjach, nie na DTO!

Reply:
ok, changed the repository to work on entities